src/cr/cube/crunch_cube.py

```diff
@@ -9,17 +9,17 @@
 from __future__ import division
 
 import json
-
 import numpy as np
+
 from scipy.stats import norm
 from scipy.stats.contingency import expected_freq
 
-from .mixins.data_table import DataTable
-from .cube_slice import CubeSlice
-from .measures.index import Index
-from .measures.scale_means import ScaleMeans
-from .utils import lazyproperty
-from . import ITEM_DIMENSION_TYPES
+from cr.cube import ITEM_DIMENSION_TYPES
```

Comment:
Ok, I checked with Bert, he says it's about personal preference. I find the argumentation to avoid relative imports compelling, mostly because they're ambiguous (without mental calculation) and more difficult to read (as a human).